app/controllers/game_invitations_controller.rb

```diff
@@ -0,0 +1,65 @@
+class GameInvitationsController < ApplicationController
+
+
+  before_action :require_user, only: %i[join create]
+
+  before_action :load_game, only: %i[create show edit]
+
+  before_action :check_user_is_author?, only: %i[create]
+
+  def show
+  end
+  
+  #user joins the game
+  def edit 
+    @current_user ||= User.find(cookies.encrypted[:user_id]) if cookies.encrypted[:user_id]
+    if !!current_user
```

Comment:
`if !!current_user` is functionally the same as `current_user`, but we have an even better method: `if logged_in?`.